addon/components/cf-navigation.js

```diff
@@ -6,12 +6,46 @@ import { ComponentQueryManager } from "ember-apollo-client";
 import { task } from "ember-concurrency";
 import getNavigationQuery from "ember-caluma/gql/queries/get-navigation";
 
+/**
+ * Component to display a nested form including navigation.
+ *
+ * ```hbs
+ * {{cf-navigation documentId="myDocId" section=section subSection=subSection}}
+ * ```
+ *
+ * @class CfFormComponent
+ */
 export default Component.extend(ComponentQueryManager, {
   layout,
   documentStore: service(),
 
+  /**
+   * The ID of the nested document to display the navigation for
+   * @argument {String} documentId
+   */
   documentId: null,
-  activeDocumentId: null,
```

Comment:
Will this be a breaking change?

Reply:
Also, what is this even? :joy:

Reply:
This is just a left-over from a previous implementation - I just cleaned it up :wink: